backend/control_center/odl/views.py

```
@@ -59,30 +59,30 @@
+            logger.debug(f"Active model {active_model} already loaded")
     else:
         logger.warning("No active model configured in model manager")
         # Try to restore from database
-        from classifier.models import ModelConfiguration
+        
         try:
             active_model_config = ModelConfiguration.objects.filter(is_active=True).first()
             if active_model_config:
-                logger.info(f"Found active model in database: {active_model_config.name}")
+                logger.debug(f"Found active model in database: {active_model_config.name}")
                 state_manager.set_active_model(active_model_config.name)
```

Ruff (0.14.2)

Undefined name `state_manager`

(F821)